sorting/selection_sort_recursive.cpp

```diff
@@ -52,7 +52,8 @@ namespace selection_sort_recursive {
  * @returns index of the minimum element
  */
 template <typename T>
-uint64_t findMinIndex(const std::vector<T> &in_arr, uint64_t current_position = 0) {
+uint64_t findMinIndex(const std::vector<T> &in_arr,
+                      uint64_t current_position = 0) {
```

Comment:
This file is not related to this PR.

Reply:
This is modified by the CI automatically; we can't do anything about it. 🙂

Reply:
This should not happen, When this file was merged why did CI not reformatted this at that time.

Reply:
> This should not happen, When this file was merged why did CI not reformatted this at that time.

That's due to the branch protection.